install/installer/pkg/components/proxy/service_test.go

```diff
@@ -27,13 +27,28 @@ func TestServiceLoadBalancerIP(t *testing.T) {
 	require.Equal(t, loadBalancerIP, svc.Spec.LoadBalancerIP)
 }
 
-func renderContextWithLoadBalancerIP(t *testing.T, loadBalancerIp string) *common.RenderContext {
+func TestServiceAnnotations(t *testing.T) {
+	annotations := map[string]string{"hello": "world"}
+
+	ctx := renderContextWithProxyConfig(t, &experimental.ProxyConfig{ServiceAnnotations: annotations})
+
+	objects, err := service(ctx)
+	require.NoError(t, err)
+
+	require.Len(t, objects, 1, "must render only one object")
+
+	svc := objects[0].(*corev1.Service)
+	for k, v := range annotations {
```

Comment:
Should be possible to use `require.EqualValues(t, annotations, svc.Annotations)`

Reply:
What we are checking here is slightly different; we need to check that the extra annotations are a *subset* of those on the rendered service. The installer hard-codes a couple of annotations too.